src/components/select-field/BaseSelectField.js

```diff
@@ -98,7 +98,14 @@ class BaseSelectField extends React.Component<Props, State> {
         // @NOTE: breaks encapsulation but alternative is unknown child ref
         const itemEl = id ? document.getElementById(id) : null;
         if (itemEl) {
-            scrollIntoViewIfNeeded(itemEl, false);
+            let parentEl = itemEl;
```

Comment:
I would look at if there are other available DOM functions to achieve something similar, like https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

Also, it seems SelectorDropDown.js has a very similar method. I don't think we should be replicating more code like that. What you can do is create a function inside utils/dom.js and extract this function in there and then use that function here as well as in SelectorDropDown.js.

Also does it have to check similar classes in both components? If not, the classes can be passed as an array of string to that function. Also ideally classes should be constants exported from their parent function. So that 'overlay-wrapper' isn't floating around in places.

Lastly I would make this change in a seperate PR as its not really related to query-bar.

Reply:
I'll move to a separate PR